README.md

```diff
@@ -7,916 +7,62 @@
 
 _Take control of your internal daemons!_
 
-**Pebble** helps you to orchestrate a set of local service processes as an organized set.
-It resembles well known tools such as _supervisord_, _runit_, or _s6_, in that it can
-easily manage non-system processes independently from the system services, but it was
-designed with unique features that help with more specific use cases.
+**Pebble** is a lightweight Linux service manager that helps you orchestrate a set of local service processes as an organized set. It resembles well known tools such as _supervisord_, _runit_, or _s6_, in that it can easily manage non-system processes independently from the system services, but it was designed with unique features that help with more specific use cases.
 
-  - [General model](#general-model)
-  - [Layer configuration examples](#layer-configuration-examples)
-  - [Using Pebble](#using-pebble)
-  - [Container usage](#container-usage)
-  - [Layer specification](#layer-specification)
-  - [API and clients](#api-and-clients)
-  - [Hacking / Development](#hacking--development)
-  - [Contributing](#contributing)
+Pebble's key features:
 
-## General model
+- Service management with [layers](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/layers/)
```

Comment:
Does the publishing system provide any way for these to be local links here and transformed into full (or rtd-local) ones when it's published? That would be nicer than having full ones here, and also avoid the need to adjust them when there's a nicer domain than `canonical-pebble.readthedocs-hosted.com`.

Reply:
What do you mean by "the publishing system"? Isn't the main place this lives the github.com repo page?

Reply:
> What do you mean by "the publishing system"? Isn't the main place this lives the github.com repo page?

Ah, I was thinking that this also ended up somewhere on the docs site, probably mixing it up in my mind with the way that PyPI has the ops README.

So irrelevant, sorry.

Reply:
I'm also asking on `~Documentation` and `~IS` whether we can get something like `pebble.canonical.com`.

Reply:
It will actually show up on https://pkg.go.dev/github.com/canonical/pebble, but I don't think many people will be reading it there (you'd have to be looking for the [Go client](https://pkg.go.dev/github.com/canonical/pebble@v1.15.0/client)).